src/features/mentee-dashboard/services/updateApplication.ts

```
@@ -0,0 +1,65 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+export interface UpdateApplicationRequestBody {
+    companyName: string;
+    applyPosition: string;
+    deadline: string;
+    location: string;
+    employmentType: string;
+    careerRequirement: number;
+}
+
+export interface UpdateApplicationResponse {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)